cmake/defaults/CXXhelpers.cmake

```diff
@@ -13,8 +13,21 @@ function(_disable_warning flag)
     set(_LAB_CXX_WARNING_FLAGS ${_LAB_CXX_WARNING_FLAGS} PARENT_SCOPE)
 endfunction()
 
+function(_set_Cxx17 proj)
+	target_compile_features(${proj} INTERFACE cxx_std_17)
+endfunction()
+
 function(_set_compile_options proj)
     if (WIN32)
         target_compile_options(${proj} PRIVATE /arch:AVX /Zi )
     endif()
+    if (UNIX)
+        # target_link_libraries(${proj} pthread jack)
+        target_link_libraries(${proj} pthread asound)
+        # target_link_libraries(${proj} pthread pulse pulse-simple)
+        target_compile_options(${proj} PRIVATE -fPIC)
+        # target_compile_definitions(${proj} PRIVATE HAVE_STDINT_H=1 HAVE_SETENV=1 HAVE_SINF=1 WEBAUDIO_KISSFFT=1 __UNIX_JACK__=1)
+        target_compile_definitions(${proj} PRIVATE HAVE_STDINT_H=1 HAVE_SETENV=1 HAVE_SINF=1 WEBAUDIO_KISSFFT=1 __LINUX_ALSA__=1)
```

Comment:
I think the only preprocessor used in this list is `WEBAUDIO_KISSFFT`

Reply:
libnyquist is sensitive to all the HAVE_*. They should be set in the libnyquist find script...